python/pyspark/broadcast.py

```diff
@@ -163,8 +163,7 @@ def destroy(self):
         """
         Destroy all data and metadata related to this broadcast variable.
         Use this with caution; once a broadcast variable has been destroyed,
-        it cannot be used again. This method blocks until destroy has
-        completed.
+        it cannot be used again.
```

Comment:
We don't need to support blocking destroy for broadcasts in Python?
It seems the test below has a behaviour change:
```
./python/pyspark/tests/test_rdd.py:327:        bdata.destroy()
```